node/src/chain_spec.rs

```diff
@@ -604,11 +608,13 @@ fn amplitude_genesis(
 		nomination: amplitude_runtime::NominationConfig { is_nomination_enabled: false },
 		dia_oracle_module: amplitude_runtime::DiaOracleModuleConfig {
 			authorized_accounts: authorized_oracles,
-			supported_currencies: vec![foucoco_runtime::AssetId::new(
-				b"Polkadot".to_vec(),
-				b"DOT".to_vec(),
-			)],
-			batching_api: b"http://dia-00.pendulumchain.tech:8070/currencies".to_vec(),
+			supported_currencies: vec![
+				foucoco_runtime::AssetId::new(b"Amplitude".to_vec(), b"AMPE".to_vec()),
+				foucoco_runtime::AssetId::new(b"Kusama".to_vec(), b"KSM".to_vec()),
+				foucoco_runtime::AssetId::new(b"Stellar".to_vec(), b"XLM".to_vec()),
+				foucoco_runtime::AssetId::new(b"FIAT".to_vec(), b"USD-USD".to_vec()),
```

Comment:
Nit: I messed this up, let's use the proper imports
```suggestion
				amplitude_runtime::AssetId::new(b"Amplitude".to_vec(), b"AMPE".to_vec()),
				amplitude_runtime::AssetId::new(b"Kusama".to_vec(), b"KSM".to_vec()),
				ampltiude_runtime::AssetId::new(b"Stellar".to_vec(), b"XLM".to_vec()),
				amplitude_runtime::AssetId::new(b"FIAT".to_vec(), b"USD-USD".to_vec()),
```